src/main/java/pages/ListingListPage.java

```diff
@@ -0,0 +1,26 @@
+package pages;
+import features.Listing;
+import features.Search;
+
+import java.util.List;
+
+public class ListingListPage {
+    private List<Listing> listings;
+
```

Comment:
Simple Javadoc documentation for the methods below if it's not too inconvenient.